libcipher/utils.py

```diff
@@ -0,0 +1,63 @@
+import string
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is English or not.
+       Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file, an exhaustive list of english words
+    """
+    
+    def __init__(self,filepath):
+       
+        self.LETTERS_AND_SPACE = string.ascii_letters + string.whitespace 
+        filename = open(filepath)
+        self.english_words = {}
+        for word in filename:
+            word = word.rstrip('\n')
+            self.english_words[word] = None
+        filename.close()
+        self.ENGLISH_WORDS = self.english_words
+        
+
+    def get_english_count(self, message):
+        #print (self.english_words)
```

Comment:
Before you push code, make sure you remove such statements which you'd have used to debug :)